procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        if tmp.statusCode != 0:
</pre><p>Instead of&nbsp;<code>if ...: raise</code>, you can do:</p><pre class="ql-syntax" spellcheck="false">assert mp.statusCode != 0, Ingestion was not successful!'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>